TransactionService/docker_context/TransactionServiceApi.py

```diff
@@ -10,81 +9,92 @@
 
 
 @app.route(endpoints.quote_endpoint, methods=['GET'])
-async def getQuote(request, trans_num, user_id, stock_symbol):
-    data = {
-        'transaction_num': trans_num,
-        'user_id': user_id,
-        'stock_symbol': stock_symbol
-    }
+async def getQuote(request):
+    data = request.json

```

Comment:
you're going to have to revert this change back along with the getQuote method signature. This is a GET HTTP request and won't have a reqest.json body

Reply:
I believe this is fixed. I wasn't putting 2 and 2 together of why everything was being passed like this. This makes sense.